lib/array_intersection.rb

```diff
@@ -1,3 +1,11 @@
 def intersection(list1, list2)
-  raise NotImplementedError, "Intersection not implemented"
+  intersection = []
+
+  list1.each do |num|
+    if list2.include?(num)
+      intersection.push(num)
+    end
+  end
+
+  return intersection
 end
```

Comment:
This works, but it's O(n * m) where n and m are the lengths of the arrays.  Can you think of an O(n + m) solution... maybe one that uses a hash?

Reply:
I believe I fixed this. What is the proper way to send it back to you, if needed?